contentcuration/contentcuration/view/registration_views.py

```diff
@@ -31,18 +31,21 @@ def send_invitation_email(request):
         try:
             user_email = data["user_email"]
             channel_id = data["channel_id"]
+            share_mode = data["share_mode"]
             retrieved_user = User.objects.get_or_create(email = user_email)
             recipient = retrieved_user[0]
             channel = Channel.objects.get(id=channel_id)
             invitation = Invitation.objects.get_or_create(invited = recipient,
                                                         email = user_email,
+                                                        share_mode=share_mode,
                                                         sender=request.user,
                                                         channel_id = channel_id,
                                                         first_name=recipient.first_name if recipient.is_active else "Guest",
                                                         last_name=recipient.last_name if recipient.is_active else " ")[0]
             ctx_dict = {    'sender' : request.user,
                             'site' : get_current_site(request),
                             'user' : recipient,
+                            'share_mode' : share_mode,
```

Comment:
Should probably wrap `share_mode` in `_(share_mode)` to ensure translation for the email message.